OpenRA.Game/Network/UnitOrders.cs

```diff
@@ -18,6 +18,8 @@ namespace OpenRA.Network
 {
 	static class UnitOrders
 	{
+		private const string ServerChatName = "Battlefield Control";
```

Comment:
I didn't know/see [Translate] tag so far. I think it is not needed here as no chat message use [Translate].